### DISCUSSION TITLE
请问为什么要移除对 Koolshare merlin 的支持？

### DISCUSSION THREAD

**Discussion by LouisLoong** [messense/aliyundrive-webdav#768] [Q&A] [answered]
如题，感觉类似我用这个系统的人挺多的，为什么呢？
还有别的解决方案吗？谢谢

**Comment by messense** [comment, marked as answer]
很简单，因为我不用了。

**Reply by LouisLoong** [reply]
真是无法反驳的理由啊 :(
我看了下好像支持也不难，也不用经常维护那一块代码的样子，可惜自己不太懂，唉。。。

**Reply by jxdwinter** [reply]
还请大佬支持下merlin吧

**Comment by chestnut1988** [comment]
附议，希望大佬能支持merlin

**Reply by baymax222** [reply]
刚想提问，大佬不用每个版本都更新，就关键的更新下呗，感谢啦，alist的也行就是不稳定，你这个比较稳定

**Comment by messense** [comment]
发起了一个梅林固件支持的投票：https://github.com/messense/aliyundrive-webdav/discussions/778

**Reply by LouisLoong** [reply]
谢谢大佬。早知道等几天了。之前因为你说不会支持了，改用AList了

**Reply by messense** [reply]
No problem, #778 可以看出来用户并不多，真正付费支持的更少。
